doc/README.hi.md

```diff
@@ -0,0 +1,143 @@
+
+### सुविधाएँ
+
+* वीडियो खोजें
+* लॉग-इन करने की ज़रूरत नहीं
+* वीडियो के बारे में साधारण जानकारी पाएँ
+* YouTube के वीडियो देखें
+* YouTube के वीडियो सुनें
+* पॉपअप मोड (फ़्लोटिंग प्लेयर)
+* वीडियो देखने के लिए स्ट्रीम करने का प्लेयर चुनें
+* वीडियो डाउनलोड करें
+* सिर्फ ऑडियो डाउनलोड करें
+* Kodi में वीडियो को खोलें
+* अगले/संबंधित वीडियो देखें
+* YouTube को किसी विशिष्ट भाषा में खोजें
+* उम्र से जुड़ी पाबंदी वाली सामग्री देखें/रोकें
+* चैनलों के बारे में साधारण जानकारी पाएँ
+* चैनल खोजें
+* किसी विशिष्ट चैनल से वीडियो देखें
+* Orbot/Tor का समर्थन (सीधे नहीं)
+* 1080p/2K/4K का समर्थन
+* देखे गए वीडियो देखें
+* चैनलों पर सदस्यता लें
+* देखे गए वीडियो खोजें
+* प्लेलिस्ट्स खोजें/देखें
+* प्लेलिस्ट्स को सूची में जोड़कर देखें
+* वीडियों को सूची में डालें
+* लोकल प्लेलिस्ट्स
+* उपशीर्षक
+* लाइवस्ट्रीम का समर्थन
+* टिप्पणियाँ देखें
+
+### समर्थित सेवाएँ
+
+NewPipe पर कई सेवाएँ समर्थित हैं। हमारे [प्रलेख](https://teamnewpipe.github.io/documentation/) में यह बताया गया है कि ऐप और एक्सट्रैक्टर में एक नई सेवा कैसे जोड़ी जा सकती है। अगर आप कोई नई सेवा जोड़ना चाहते हैं, कृपया हमसे संपर्क करें। वर्तमान में समर्थित सेवाएँ हैं:
+
+* YouTube
+* SoundCloud \[बीटा\]
+* media.ccc.de \[बीटा\]
+* PeerTube instances \[बीटा\]
+* Bandcamp \[बीटा\]
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
+## स्थापना और अपडेट्स
+आप इनमें से किसी एक तरीके से NewPipe को स्थापित कर सकते हैं:
+ 1. हमारे अनुकूलित रिपॉज़िटरी को F-Droid पर जोड़ें और उसे वहाँ से स्थापित करें। अनुदेश यहाँ हैं: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
+ 2. [GitHub प्रकाशन](https://github.com/TeamNewPipe/NewPipe/releases) से APK डाउनलोड करें और उसे स्थापित करें।
+ 3. F-Droid के ज़रिए अपडेट करें। यह अपडेट करने का सबसे धीमा तरीका है, क्योंकि पहले F-Droid को बदलाव पहचानने होंगे, फिर यह APK बनाएगा, उसपर हस्ताक्षर बनाएगा, और आखिर में सदस्यों को अपडेट पुश करेगा।
+ 4. अपने आप एक डीबग APK बनाएँ। यह अपडेट पाने का सबसे तेज़ तरीका है, मगर यह काफ़ी कठिन है तो हम सलाह देंगे कि आप किसी दूसरे तरीके का इस्तेमाल करें।
+ 
+ज़्यादातर सदस्यों के लिए पहला तरीका ठीक है। पहले और दूसरे तरीके से स्थापित APK एक-दूसरे के अनुकूल हैं, मगर तीसरे तरीके से स्थापित APK से नहीं। ऐसा इसलिए है क्योंकि पहले और दूसरे तरीके में एक ही (हमारे) हस्ताक्षर कुँजी का इस्तेमाल हुआ है, मगर तीसरे तरीके में दूसरे (F-Droid के) हस्ताक्षर कुँजी का इस्तेमाल किया जाता है। चौथे तरीके से अपना डीबग APK फ़ाइल बनाने पर कुँजी की ज़रूरत ही नहीं पड़ती। कुँजियों से यह निश्चित किया जाता है कि सदस्य कहीं ऐप पर कोई दुर्भावनापूर्ण अपडेट तो स्थापित नहीं कर रहा।
+
+इस दौरान अगर आप स्रोत बदलना चाहते हैं (मान लीजिए NewPipe का मूल हिस्सा टूट गया है और F-Droid पर अब भी नवीनतम संस्करण नहीं है), हम इस तरीके का इस्तेमाल करने की सलाह देंगे:
+1. सेटिंग्स > सामग्री > डेटाबेस निर्यात करें के ज़रिए अपने डेटा को बैकअप कर लें ताकि आपके पास अपने देखे गए वीडियो, सदस्यताएँ और प्लेलिस्ट्स हो

```

Comment:
"database niryaat karein. Iske zariye" start a new sentence, to avoid confusion, or put the settings titles in code blocks.